application/models/Entity/Instance.php

```diff
@@ -1458,4 +1458,8 @@ public function getInterfaceVersion()
     {
         return $this->interfaceVersion;
     }
+
+    public function isUsingVueUI() {
```

Comment:
Same

Reply:
Alternatively, make this a helper method, or set a flag on the Instance_controller superclass.

Reply:
moved to `instance_helper`